Mathlib/Tactic/TryThis.lean

```diff
@@ -44,7 +44,7 @@ def addRewriteSuggestion (origTac : Syntax) (e : Expr) (symm : Bool) (type? : Op
   -- We resort to using `logInfoAt` here rather than `addSuggestion`,
   -- as I've never worked out how to have `addSuggestion` render comments.
   if let some type := type? then
-    logInfoAt origTac m!"{tac}\n-- {← ppExpr type}"
+    logInfoAt origTac m!"{tac}\n-- {← PrettyPrinter.ppExpr type}"
```

Comment:
This file needs to be rewritten to use the new TryThis API, it seems to be largely a reimplementation of the stub

Reply:
Is it okay if I do this in a separate PR? I would like to work on lots of the tactics that can take advantage of the new Try this functionality, but I would like to get it available for everyone asap without holding up on rewriting old stuff.

Reply:
Thanks!